code/modules/research/designs/medical_designs.dm

```diff
@@ -54,6 +54,16 @@
 	category = list("Medical Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL | DEPARTMENTAL_FLAG_SCIENCE
 
+/datum/design/cloning_disk
+	name = "Cloning Data Disk"
+	desc = "Produce additional disks for storing genetic data."
+	id = "cloning_disk"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 300, MAT_GLASS = 100, MAT_SILVER=50)
+	build_path = /obj/item/disk/data
+	category = list("Medical Designs")
+	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL
```

Comment:
give to sci too, genetics is owned by both.

Reply:
Will do.